verovio-util/verovio-proxy.js

```diff
@@ -4,200 +4,258 @@ verovio.vrvToolkit = verovio.vrvToolkit || {};
 
 // Constructor and destructor
 // Toolkit *constructor()
-verovio.vrvToolkit.constructor = Module.cwrap('vrvToolkit_constructor', 'number', []);
+verovio.vrvToolkit.constructor = Module.cwrap( 'vrvToolkit_constructor', 'number', [] );
 
 // void destructor(Toolkit *ic)
-verovio.vrvToolkit.destructor = Module.cwrap('vrvToolkit_destructor', null, ['number']);
+verovio.vrvToolkit.destructor = Module.cwrap( 'vrvToolkit_destructor', null, ['number'] );
 
 // bool edit(Toolkit *ic, const char *editorAction) 
-verovio.vrvToolkit.edit = Module.cwrap('vrvToolkit_edit', 'number', ['number', 'string']);
+verovio.vrvToolkit.edit = Module.cwrap( 'vrvToolkit_edit', 'number', ['number', 'string'] );
 
 // char *editInfo(Toolkit *ic)
-verovio.vrvToolkit.editInfo = Module.cwrap('vrvToolkit_editInfo', 'string', ['number']);
+verovio.vrvToolkit.editInfo = Module.cwrap( 'vrvToolkit_editInfo', 'string', ['number'] );
 
 // char *getAvailableOptions(Toolkit *ic)
-verovio.vrvToolkit.getAvailableOptions = Module.cwrap('vrvToolkit_getAvailableOptions', 'string', ['number']);
+verovio.vrvToolkit.getAvailableOptions = Module.cwrap( 'vrvToolkit_getAvailableOptions', 'string', ['number'] );
 
 // char *getElementAttr(Toolkit *ic, const char *xmlId)
-verovio.vrvToolkit.getElementAttr = Module.cwrap('vrvToolkit_getElementAttr', 'string', ['number', 'string']);
+verovio.vrvToolkit.getElementAttr = Module.cwrap( 'vrvToolkit_getElementAttr', 'string', ['number', 'string'] );
 
 // char *getElementsAtTime(Toolkit *ic, int time)
-verovio.vrvToolkit.getElementsAtTime = Module.cwrap('vrvToolkit_getElementsAtTime', 'string', ['number', 'number']);
+verovio.vrvToolkit.getElementsAtTime = Module.cwrap( 'vrvToolkit_getElementsAtTime', 'string', ['number', 'number'] );
+
+// char *vrvToolkit_getExpansionIdsForElement(Toolkit *tk, const char *xmlId);
+verovio.vrvToolkit.getExpansionIdsForElement = Module.cwrap( 'vrvToolkit_getExpansionIdsForElement', 'string', ['number', 'string'] );
 
 // char *getHumdrum(Toolkit *ic)
-verovio.vrvToolkit.getHumdrum = Module.cwrap('vrvToolkit_getHumdrum', 'string');
+verovio.vrvToolkit.getHumdrum = Module.cwrap( 'vrvToolkit_getHumdrum', 'string' );
 
 // char *getLog(Toolkit *ic)
-verovio.vrvToolkit.getLog = Module.cwrap('vrvToolkit_getLog', 'string', ['number']);
+verovio.vrvToolkit.getLog = Module.cwrap( 'vrvToolkit_getLog', 'string', ['number'] );
+
+// char *getMEI(Toolkit *ic, const char *options)
+verovio.vrvToolkit.getMEI = Module.cwrap( 'vrvToolkit_getMEI', 'string', ['number', 'string'] );
 
-// char *getMEI(Toolkit *ic, int pageNo, int scoreBased)
-verovio.vrvToolkit.getMEI = Module.cwrap('vrvToolkit_getMEI', 'string', ['number', 'number', 'number']);
+// char *vrvToolkit_getNotatedIdForElement(Toolkit *tk, const char *xmlId);
+verovio.vrvToolkit.getNotatedIdForElement = Module.cwrap( 'vrvToolkit_getNotatedIdForElement', 'string', ['number', 'string'] );
```

Comment:
What do the comments with a C/C++ function mean? Is it the function that this code will call in the backend?

Reply:
Exactly. The cwrap stuff is pretty unintelligible, so this is just here to show what values are expected in the C++ functions.